doc/user/content/ingest-data/network-security/static-ips.md

```diff
@@ -31,7 +31,24 @@ connections from your region's egress IP addresses. You must allow connections
 from all egress IP addresses associated with your region. Connections may
 originate from any one of the egress IP addresses.
 
-To find the egress IP addresses associated with your region, you can query the
+## AWS Static Ips
+
+Each region of materialize uses a static set of CIDRs for egress IPs.
```

Comment:
This is stated above, so not sure we need this. Maybe something like, "To allow Materialize to connect, please allowlist your region's egress CIDRs in your firewall" or something like that